server/repository/src/migrations/mod.rs

```diff
@@ -146,7 +147,8 @@ pub fn migrate(
         Box::new(v2_08_00::V2_08_00),
         Box::new(v2_08_03::V2_08_03),
         Box::new(v2_09_00::V2_09_00),
-        Box::new(v2_10_00::V2_10_00),
+        Box::new(v2_09_01::V2_09_01),
+        // Box::new(v2_10_00::V2_10_00),
```

Comment:
These causing tests to fail... they're not supposed to be in 2.9.1 branch. Ferg has PR to remove them.

Reply:
Oof lol, has that gone in now?